### PR TITLE
Paginate assessment lists

### PR DESCRIPTION
**Pull request by munnsmunns**
Add pagination to the public assessment list and the user's assessment list. Each page displays 50 assessments.

### CODE REVIEW

File: hawc/apps/assessment/forms.py [hunk 1]
```diff
@@ -143,6 +143,12 @@ def helper(self):
         helper.add_row("search", 2, ["col-md-8", "col-md-4"])
         return helper
 
+    def clean_order_by(self):
+        order_by = self.cleaned_data["order_by"]
+        if order_by == "":
+            return "-last_updated"
```
Comment (munnsmunns) [inline hawc/apps/assessment/forms.py:149]: This fixes a bug that when going to the second page, order_by becomes an empty string if the user doesn't click "Apply Filters" while on the first page. Perhaps there's another way to fix this but this seems to be the most simple

**Review by shapiromatron — APPROVED** [review]
i accidentally dreamed code last night after our chat thinking about this one.  Here's my main change:

```python
def get_order_by(self):
    value = self.cleaned_data.get("order_by") if self.is_valid() else None
    return value or self.DEFAULT_ORDER_BY
```

The key here is an empty string is falsey, so it evaluates the OR and therefore the default order.

Also cleaned up some unused code and simplified the list view. Feel free to merge if the changes work and you're happy with the revisions.

File: hawc/apps/assessment/forms.py (hunk 1 above)
Comment (shapiromatron) [reply hawc/apps/assessment/forms.py]: Comments were interesting on this one, wow the initial vs default isn't what I expected. https://www.peterbe.com/plog/initial-values-bound-django-form-rendered

Comment (munnsmunns) [reply hawc/apps/assessment/forms.py]: Yeah it is weird that there is no default option

**Pull request merged**